src/routers/user.ts

```diff
@@ -1,24 +1,50 @@
 import express from 'express';
+import { create, read, update, del } from '../operators/UserOperations';
 
 const userRouter = express.Router();
 
-userRouter.get('/', (req, res) => {
-	console.log('Get User');
-	res.send('Get userRouter');
+userRouter.get('/:id', async (req, res) => {
+	try {
+		console.log('Get User');
+		const user = await read(req.params.id);
+		res.send({ ...user, semesters: Array.from(user?.semesters || []) });
+	} catch (e) {
+		res.status(404).send(e);
```

Comment:
Let's define a constant error message like `const userError = "User not found.";` then instead of `res.send(e)` do `res.send(userError)`.

We don't want people to know why their request failed because it is a security risk, and if we program our frontend properly, a regular user should not encounter this kind of error often, or at all.